### PR TITLE
feat: Mainpage Responsive Web Develop

### PR DESCRIPTION
**Pull request by juntaepark**

### CODE REVIEW

**Comment by jinoosss** [comment]
Nicely done. 
There seems to be a need for a way to handle window size globally. (Considering rendering with it)
This could be a good next step.

**Review by jinoosss — APPROVED** [review]
LGTM !

File: packages/web/src/components/common/search-input/SearchInput.tsx [hunk 1]
```diff
@@ -29,7 +29,9 @@ const SearchInput: React.FC<SearchInputProps> = ({
       className={cx(className, { "empty-status": value === "" })}
     >
       <InputStyle placeholder={placeholder} value={value} onChange={onChange} />
-      <IconSearch className="search-icon" />
+      <div>
```
[the remark this answers is not in the document]
Comment (juntaepark) [reply packages/web/src/components/common/search-input/SearchInput.tsx]: Thank you for your confirmation.

File: packages/web/src/styles/globalStyle.ts [hunk 2]
```diff
@@ -17,6 +18,12 @@ const globalStyle = (theme: Theme) => css`
   body {
     background-color: ${theme.color.background01};
     padding-top: 70px;
+    ${media.tablet} {
+      padding-top: 60px;
+    }
+    ${media.mobile} {
+      padding-top: 50px;
```
[the remark this answers is not in the document]
Comment (juntaepark) [reply packages/web/src/styles/globalStyle.ts:25]: footer was covered and padded.

**Pull request merged**